src/rdkafka_request.c

```diff
@@ -2354,12 +2373,13 @@ rd_kafka_resp_err_t rd_kafka_MetadataRequest(rd_kafka_broker_t *rkb,
          * and should go before most other requests (Produce, Fetch, etc). */
         rkbuf->rkbuf_prio = RD_KAFKA_PRIO_HIGH;
 
-        rd_kafka_broker_buf_enq_replyq(rkb, rkbuf,
-                                       /* Handle response thru rk_ops,
-                                        * but forward parsed result to
-                                        * rko's replyq when done. */
-                                       RD_KAFKA_REPLYQ(rkb->rkb_rk->rk_ops, 0),
-                                       rd_kafka_handle_Metadata, rko);
+        rd_kafka_broker_buf_enq_replyq(
+            rkb, rkbuf,
+            /* Handle response thru rk_ops,
+             * but forward parsed result to
+             * rko's replyq when done. */
+            RD_KAFKA_REPLYQ(rkb->rkb_rk->rk_ops, 0),
```

Comment:
I would suggest to add more information regarding `rko` and `resp_cb`. When what is used so that in future its easier to read.